dds/DCPS/DomainParticipantFactoryImpl.cpp

```diff
@@ -90,6 +90,15 @@ DomainParticipantFactoryImpl::create_participant(
     this, domainId, value.id, par_qos, a_listener, mask, value.federated
   );
 
+  if (DCPS_debug_level > 0) {
```

Comment:
This should probably be a higher debug level.  See section 7.5.1 of the Developer's Guide.